content/geoip/release-notes/2023.mdx

```diff
@@ -7,6 +7,26 @@ title: GeoIP2 Release Notes
 Subscribe to the [GeoIP2 release notes RSS feed](/geoip/release-notes/rss.xml).
 </Alert>
 
+<ReleaseNote date="2023-08-28" title="New Satellite connection type releasing">
+
+Tomrrow, on Tuesday August 29, 2023 we will be adding a new connection type
+value, `Satellite`, which will identify satellite internet providers such as
+Starlink, Viasat, and Hughes. Satellite connections typically have higher
+latency than cable/DSL connections.
+
+The `Satellite` connection type value will appear in the following databases:
+
+- GeoIP Connection Type database
+- GeoIP Enterprise database
+
+Users of the following client APIs will need to update their API to the latest
+version to avoid breaking changes with the new value:
+
+- [Update Java API to version 4.1.0](https://central.sonatype.com/search?q=g:com.maxmind.geoip2%20%20a:geoip2&smo=true)
```

Comment:
```suggestion
- [Update Java API to version 4.1.0](https://central.sonatype.com/artifact/com.maxmind.geoip2/geoip2/4.1.0)
```

Reply:
https://central.sonatype.com/artifact/com.maxmind.geoip2/geoip2/4.1.0 takes a user directly to the specified version and loads a lot faster for some reason. But if it just needs to be at least version 4.1.0 (e.g. eventually 4.2.0 will work as well), then maybe it might be worth including something like "Update to at least..."